tests/modules/ecs6-class/line.test.js

```diff
@@ -0,0 +1,177 @@
+const { Line, getPointByX, getPointByY } = require('../../../modules/ecs6-class/line');
+const { Point } = require('../../../modules/ecs6-class/point');
+const mockConstructorLine = jest.fn(constructor)
+const mockConstructorPoint = jest.fn(constructor)
+
+
+describe('CONSTRUCTOR', () => {
+
+    it('should initialize x and y to default values when no parameters are provided', () => {
+
```

Comment:
don't write all test cases in one block, split them according to their test case
and write a good sentence that explains the test case